lib/workers/branch/lock-files.js

```diff
@@ -123,6 +123,41 @@ async function writeExistingFiles(config) {
         upath.join(basedir, 'package.json'),
         JSON.stringify(massagedFile)
       );
+
+      // if you have some reference to local files, copy them
+      const toCopy = listLocalLibs(massagedFile.dependencies);
+      toCopy.push(...listLocalLibs(massagedFile.devDependencies));
+      if (toCopy.length !== 0) {
+        logger.debug(`listOfNeededLocalFiles files to copy: ${toCopy}`);
+        await Promise.all(
+          toCopy.map(async localPath => {
+            const resolvedLocalPath = path.resolve(basedir, localPath);
+            if (!resolvedLocalPath.startsWith(config.tmpDir.path)) {
+              logger.info(
+                `local lib '${
+                  toCopy
+                }' will not be copied because it's out of the repo.`
+              );
+            }
+            // at the root of local Lib we should find a package.json and yarn will use it to update yarn.lock file
```

Comment:
Comment maybe doesn't need to be yarn-specific? I assume same goes for `npm`?

Reply:
Yes, it should work with npm also.
But I didn't test.

I update the comment